Jax/Database.php

```
@@ -99,14 +95,9 @@ public function ftable(string $tableName): string
         return '`' . $this->prefix . $tableName . '`';
     }
 
-    public function error(): string
-    {
-        return $this->mysqli->error;
-    }
```

Is there some exception handling that needs to be added to handle errors now or are you just going to let it 500?

---

I'm just gonna let it throw exceptions. It's basically what was happening before anyway (MySQL errors printed to the page as warnings) and I'd rather errors just be loud